config/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgis
+  pool: 5
+  timeout: 5000
+  schema_search_path: "public,postgis"
+  script_dir: /usr/local/share/postgis/
+  encoding: unicode
+  username: safecast
+  password:
+
+development:
+  <<: *default
+  database: ingest-solarcast_development
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: ingest-solarcast_test
+
+production:
+  <<: *default
+  database: ingest-solarcast_production
```

Comment:
Would it be okay to define `DATABASE_URL` like `export DATABASE_URL="postgis://${PGUSER}:${PGHOST}@${PGHOST}:${PGPOR      T}/${PGDATABASE}"` in `.ebextensions/postgresql.config` instead? 

According to http://guides.rubyonrails.org/configuring.html#connection-preference ,

```
If you have both config/database.yml and ENV['DATABASE_URL'] set then Rails will merge the configuration together. To better understand this we must see some examples.

When duplicate connection information is provided the environment variable will take precedence:
```